ktlint-rule-engine-core/src/main/kotlin/com/pinterest/ktlint/rule/engine/core/internal/IdNamingPolicy.kt

```diff
@@ -3,7 +3,7 @@ package com.pinterest.ktlint.rule.engine.core.internal
 /**
  * Provides policy to have consistent and restricted `id` field naming style.
  */
-internal object IdNamingPolicy {
+public object IdNamingPolicy {
```

Comment:
The `IdNamingPolicy` may not be made public for the reason of exposing the function to check the ruleId regex. Please use try-catch solution which I proposed in other comment.

Reply:
Sure. I have removed the `public`